MC/bin/o2dpg_sim_workflow.py

```diff
@@ -928,6 +928,11 @@ def addQCPerTF(taskName, needs, readerCommand, configFilePath, objectsFile=''):
                    needs=[TOFTPCMATCHERtask['name']],
                    readerCommand='o2-global-track-cluster-reader --track-types "ITS-TPC-TOF,TPC-TOF,TPC,ITS-TPC-TRD,ITS-TPC-TRD-TOF,TPC-TRD,TPC-TRD-TOF" --cluster-types none',
                    configFilePath='json://${O2DPG_ROOT}/MC/config/QC/json/tofMatchedTracks_AllTypes_direct_MC.json')
+     ### ITS
+     addQCPerTF(taskName='ITSTrackSimTask',
+                needs=[ITSRECOtask['name']],
+                readerCommand='o2-its-reco-workflow',
```

Comment:
Should I understand this invokes reconstruction again just to provide results for QC? Can we perhaps avoid reconstructing twice?

Reply:
We should definitely not reconstruct twice, here one should use a reader (unless the its-reco is used as a reader, as TOF does - but in that case, we specify some options to use it as a reader).

Reply:
How can I remove that? I tried to leave "needs" parameter empty, but it ends up with an error.

Reply:
You need some kind of reader to provide data to QC, I don't think that putting an empty value there would make any sense. What we were suggesting was to use a file reader which just picks up the reconstructed data instead of performing reco again.

Reply:
Please, check if the `o2-global-track-cluster-reader` with the correct options (e.g. we use `--track-types "ITS-TPC-TOF,TPC-TOF,TPC,ITS-TPC-TRD,ITS-TPC-TRD-TOF,TPC-TRD,TPC-TRD-TOF" --cluster-types none'` for TOF) can be used.